src/Service/Resolvers/SqlMutationEngine.cs

```diff
@@ -40,6 +41,7 @@ public class SqlMutationEngine : IMutationEngine
         private readonly IHttpContextAccessor _httpContextAccessor;
         private readonly GQLFilterParser _gQLFilterParser;
         public const string IS_FIRST_RESULT_SET = "IsFirstResultSet";
+        private const string TRANSACTION_EXCEPTION_ERROR_MSG = "An unexpected error occured";
 
```

Comment:
This error message seems more generic than for just Transactions.

Reply:
updated